monad-bayes.cabal

```diff
@@ -122,7 +131,7 @@ executable example
 
   if flag(dev)
     ghc-options:
-      -Wall -Wcompat -Wincomplete-record-updates
+      -threaded -O2 -Wall -Wcompat -Wincomplete-record-updates
```

Comment:
Do we really need `threaded`? Maybe at some point in the future we will want to run sequential Monte Carlo in parallel or parallel chains but I don't think we do now?